aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -166,6 +166,9 @@ builder.Services.AddServerSideBlazor()
     });
 ```
 
+> [!WARNING]
+> We recommend setting <xref:Microsoft.AspNetCore.SignalR.HubOptions.MaximumReceiveMessageSize> to a small value in order to limit the risk of [Denial of service (DoS) attacks](xref:blazor/security/server/threat-mitigation#denial-of-service-dos-attacks). The default value is 32 KB (32,768 bytes).
```

Comment:
I believe 32KB should be sufficient unless we follow the convention of writing total bytes elsewhere?

```suggestion
> We recommend setting <xref:Microsoft.AspNetCore.SignalR.HubOptions.MaximumReceiveMessageSize> to a small value in order to limit the risk of [Denial of service (DoS) attacks](xref:blazor/security/server/threat-mitigation#denial-of-service-dos-attacks). The default value is 32 KB.
```

Reply:
> the convention

That's the problem. The standard was never adopted. They wrote a whole page on how to use bit/byte terms without even addressing it ...

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms

I think ur aware technically (IEC and SI) that we should be using "KiB" (kibibyte) to eliminate the confusion (or "kB" for multiples of 1,000). Since they won't permit that approach, this is how I've been hardening docs to avoid confusion.

Reply:
I might change it tho ... I'm going to open an issue to discuss this with the docs 🐈 🐈 🐈.

I performed a search to see how consistent I've been. It looks like in more than half the cases, I did write out either format (X bytes or X * 1024). It looks like a number of cases without it exist, mostly in content sent over from the PU that I didn't change on publication.

I'll circle around and update depending on the outcome of that discussion ... but let's not hold this up over that.